model_analyzer/analyzer.py

```diff
@@ -165,6 +163,8 @@ def analyze(self, mode, quiet):
         if not quiet:
             self._result_manager.write_results()
 
+        logger.info(self._get_report_command_help_string())
```

Comment:
curious if `quiet` should be honored here?

Reply:
`logger.info()` already respects the quiet option:

https://github.com/triton-inference-server/model_analyzer/blob/matthewkotila-add-next-steps-to-cli-output/model_analyzer/entrypoint.py#L246-L247

https://github.com/triton-inference-server/model_analyzer/blob/matthewkotila-add-next-steps-to-cli-output/model_analyzer/entrypoint.py#L246-L247